FirefoxLite_Downloader_Library/src/main/java/com/example/firefoxlitedownloaderlibrary/LiveDataHelper.java

```diff
@@ -4,19 +4,27 @@
 import androidx.lifecycle.MediatorLiveData;
 
 public class LiveDataHelper {
+
     private MediatorLiveData<Integer> _percent = new MediatorLiveData<>();
+    private static LiveDataHelper liveDataHelper;
```

Comment:
since this Helper is static, the liveData it holds is also static-ish scope. 
Consider putting the LiveData in a viewModel

Reply:
Will be sending a pull request for this .